bin/githooks/pre-commit

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Ensure remote is up to date
+git fetch
+
+# Check if the version has changed at all. If so, assume the wanted change has been made.
+diff=$(git diff origin/master package.json | grep '"version":')
+
+if [[ -z $diff ]]; then
+  echo "App version unchanged, bumping patch number..."
+
+  # Get the current patch number and increase by 1
+  patch_version=$(grep '"version":' package.json | sed -n 's/.*"\([0-9]*\).\([0-9]*\).\([0-9]*\).*/\3/p')
```

Comment:
We just need to be careful this is picking up the right version. Could mess up if 'version' is used in a package name.